### PR TITLE
Refactored the Sentiment Analysis tutorial to use the Cloud Client Lib

### PR DESCRIPTION
**Pull request by jmdobry**
Blocked by GoogleCloudPlatform/google-cloud-python#2858

### CODE REVIEW

**Comment by jmdobry** [comment]
I think you mean @puneith

**Comment by jmdobry** [comment]
This is unblocked and ready for final review

File: language/sentiment/sentiment_analysis.py [hunk 1]
```diff
@@ -11,36 +11,21 @@
...
-            }
-        )
-        response = service_request.execute()
-
-    score = response['documentSentiment']['score']
-    magnitude = response['documentSentiment']['magnitude']
-
-    for i, sentence in enumerate(response['sentences']):
-        sentence_sentiment = sentence['sentiment']['score']
+    for i, sentence in enumerate(annotations.sentences):
```
Comment (puneith) [inline language/sentiment/sentiment_analysis.py]: change the variable name from "i" to "index"

**Comment by puneith** [comment]
Other than the earlier comment LGTM

**Comment by jonparrott** [comment]
I'm okay with i, but "n" is idiomatic in combination with "enumerate".

**Comment by gguuss** [comment]
FYI, it looks like flake8 is not liking a couple things:

```
nox > virtualenv /home/travis/build/GoogleCloudPlatform/python-docs-samples/.nox/lint-sample-language-sentiment
nox > pip install --upgrade flake8 flake8-import-order
nox > chdir ./language/sentiment
nox > flake8 --show-source --builtin gettext --max-complexity 15 --import-order-style google --exclude .nox,.cache,env,lib,generated_pb2 --application-import-names resources,sentiment_analysis_test,sentiment_analysis .
./sentiment_analysis.py:23:1: E302 expected 2 blank lines, found 1
def print_result(annotations):
^
./sentiment_analysis.py:51:13: E128 continuation line under-indented for visual indent
            include_syntax=False, include_entities=False)
```

File: language/sentiment/sentiment_analysis.py (hunk 1 above)
Comment (jmdobry) [inline language/sentiment/sentiment_analysis.py]: Done

**Pull request merged**